app/packages/registration-schema/validations.ts

```diff
@@ -65,6 +65,40 @@ export function buildValidation(groups: SchemaBlockGroup[], node?: NodeModel) {
     return ret;
 }
 
+export function validateNodeLicense() {
```

Comment:
The validator function is async because we have to `await` for `content.license`.